CHANGELOG.unreleased.md

```diff
@@ -7,7 +7,7 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
-- [Host/Advanced] Add the field `Scheduler granularity`. Only available on XCP-ng >= 8.2 and CH >= 8.2 [#5191](https://github.com/vatesfr/xen-orchestra/issues/5291) (PR [#5320](https://github.com/vatesfr/xen-orchestra/pull/5320))
+- [Host/Advanced] Ability to change the scheduler granularity. Only available on XCP-ng >= 8.2 and CH >= 8.2 [#5191](https://github.com/vatesfr/xen-orchestra/issues/5291) (PR [#5320](https://github.com/vatesfr/xen-orchestra/pull/5320))
```

Comment:
It's not available in Citrix Hypervisor at all actually. I mean, at Xen level there's support for changing the granularity, but not in their version of the XAPI, and even when they will have a version of XAPI that supports it, they don't plan to enable it.

Reply:
Yes this is correct, CH 8.2 does NOT include the feature and it might never includes since it can be restricted via a conf file.